chia/full_node/mempool.py

```diff
@@ -95,7 +95,7 @@ def add_to_pool(self, item: MempoolItem) -> None:
         self.sorted_spends[item.fee_per_cost][item.name] = item
 
         for coin in item.removals:
-            coin_id = coin.name()
+            coin_id: bytes32 = coin.name()
```

Comment:
This patch is already very large. I think it would benefit from not also including unrelated fixups

Reply:
Thanks, done.